.github/workflows/ci.yml

```diff
@@ -92,11 +92,26 @@ jobs:
             declare -a "bors_args=(${bors_args})"
           fi
 
-          if [[ "${bors_args[0]-}" == --target ]]; then
-            echo "Only running targets matching '${bors_args[1]}'."
-            matrix="$(
-              target="${bors_args[1]}" yq --output-format json '[.[] | select(.target | test(env(target)))]' <<< "${matrix}"
-            )"
+          targets=()
+          index=0
+          while [[ "${bors_args[$index]-}" == --target ]]; do
+            index=$(expr $index + 1)
+            targets+=("${bors_args[$index]}")
+            index=$(expr $index + 1)
+          done
+
+          selects=()
+          for target in "${targets[@]}"; do
+            selects+=(".target == \"${target}\"")
+          done
+          query=$(printf " or %s" "${selects[@]}")
+          query="${query:4}"
```

Comment:
This is because the syntax is `.target == "$target1" or .target == "$target2"`, but this `printf` join has ` or ` at the start, so this indexes off the first 4 characters.

Reply:
Say I have the following:

```bash
$ selects=(...)
$ echo "${selects[@]}"
.target == "thumbv7em-none-eabihf" .target == "thumbv7m-none-eabi
$ query=$(printf " or %s" "${selects[@]}")
$ echo "${query}"
 or .target == "thumbv7em-none-eabihf" or .target == "thumbv7m-none-eabi"
$ query="${query:4}"
$ echo "${query}"
.target == "thumbv7em-none-eabihf" or .target == "thumbv7m-none-eabi"
```

So this indexing just removes the first ` or `, and this seems to be the simplest way to join an array in bash.

Reply:
ah, that makes sense, there's probably a good awk script for it but this is fine